pymeasure/instruments/hp/hp34401A.py

```diff
@@ -85,97 +98,75 @@ def __init__(self, adapter, **kwargs):
         validator=strict_discrete_set,
         values=FUNCTIONS,
         map_values=True,
-        get_process=lambda v: v.strip('"'))
+        get_process=lambda v: v.strip('"')
```

Comment:
As a suggestion: If you put the final parenthesis in a new line, it is good to end this line with a comma. That way you can add more parameters without touching the `get_process` line:
```suggestion
        get_process=lambda v: v.strip('"'),
```
Similarly below.

It is totally fine to end all, even the last, line with a comma.